### PR TITLE
Design hot fixes

### PR DESCRIPTION
**Pull request by DevStarlight**
Full redesign of [#156613344](https://www.pivotaltracker.com/story/show/156613344)

### CODE REVIEW

File: pages/video-download/video-download.view.html [hunk 1]
```diff
@@ -8,8 +8,10 @@
 	</div>
 
 	<div ng-if="VideoDownload.video.data.error && !VideoDownload.loading" class="headers-info-videoDownload">
+			<!-- TO-DO: Set translation -->
 		<h1 class="header">Video erróneo</h1>
```
Comment (CodingCarlos) [inline pages/video-download/video-download.view.html]: Don't forget translating this ;)

File: pages/video-download/video-download.view.html [hunk 2]
```diff
@@ -8,8 +8,10 @@
...
 		<div>
+			<!-- TO-DO: Set translation -->
 			No hemos podido encontrar ese video. Por favor, comprueba que el enlace es correcto, y prueba de nuevo.
```
Comment (CodingCarlos) [inline pages/video-download/video-download.view.html]: Don't forget translating this ;)

**Review by CodingCarlos — APPROVED** [review]
Remember translating that two things, and my corde review is done ;)

File: pages/video-download/video-download.view.html (hunk 1 above)
Comment (DevStarlight) [reply pages/video-download/video-download.view.html]: holy sh**, thanks!

**Pull request merged**